otter/convergence/steps.py

```diff
@@ -220,7 +220,7 @@ def as_effect(self):
             next_interval=exponential_backoff_interval(2))
 
         def report_success(result):
-            return StepResult.SUCCESS, []
+            return StepResult.RETRY, ['must re-gather after deletion']
```

Comment:
Nitpick - maybe "must re-gather after deletion in order to update the active cache" ?